$NetBSD$

Support Catch2 v3.

Remove empty binarize test which fails with Catch2 v3.8.0.

--- tests/CMakeLists.txt.orig	2024-03-28 10:51:35.000000000 +0000
+++ tests/CMakeLists.txt
@@ -1,4 +1,4 @@
-find_package(Catch2 2.9 REQUIRED)
+find_package(Catch2 REQUIRED)
 
 include(Catch)
 
@@ -16,7 +16,7 @@ file(TO_NATIVE_PATH "${TEST_DATA_DIR}" T
 add_library(test_common INTERFACE)
 target_include_directories(test_common INTERFACE ${CMAKE_CURRENT_LIST_DIR})
 target_compile_definitions(test_common INTERFACE TEST_DATA_DIR=R"\(${TEST_DATA_DIR}\)" CATCH_CONFIG_FAST_COMPILE)
-target_link_libraries(test_common INTERFACE Catch2::Catch2 Boost::nowide)
+target_link_libraries(test_common INTERFACE Catch2::Catch2WithMain Boost::nowide)
 
 if (EMSCRIPTEN)
     target_link_libraries(test_common INTERFACE nodefs.js noderawfs.js)
@@ -24,10 +24,6 @@ endif()
 
 add_subdirectory(core)
 
-if (${PROJECT_NAME}_BUILD_COMPONENT_Binarize)
-    add_subdirectory(binarize)
-endif()
-
 if (${PROJECT_NAME}_BUILD_COMPONENT_Convert)
     add_subdirectory(convert)
-endif ()
\ No newline at end of file
+endif ()
